js/components/InfoForm/index.js

```diff
@@ -0,0 +1,168 @@
+import React, { Component, PropTypes } from 'react';
+import Gandalf from 'gandalf-validator';
+import { TextInput, View, Text, Button, DatePickerIOS, TouchableHighlight } from 'react-native';
+import { styles } from './../../styles/formStyles';
+
+class Form extends Gandalf {
+  constructor() {
+    const fields = {
+      potluckName: {
+        component: TextInput,
+        validators: ['required'],
+        errorPropName: 'error',
+        onChangeHandler: 'onChangeText',
+        props: {
+          placeholder: 'Potluck Name',
+          style: styles.login
+        },
+        getValueInOnChange: (text) => text,
+        debounce: 500,
+      },
+      Theme: {
+        component: TextInput,
+        validators: ['required'],
+        errorPropName: 'error',
+        onChangeHandler: 'onChangeText',
+        props: {
+          placeholder: 'Theme',
+          style: styles.login
+        },
+        getValueInOnChange: (text) => text,
+        debounce: 500,
+      },
+      guestNumber: {
+        component: TextInput,
+        validators: ['numeric'],
+        errorPropName: 'error',
+        onChangeHandler: 'onChangeText',
+        props: {
+          placeholder: 'Number of Guests',
+          style: styles.login
+        },
+        getValueInOnChange: (text) => text,
+        debounce: 500,
+      },
+    }
+    super(fields);
+    this.state.date = new Date()
```

Comment:
Use newlines to separate concerns. the call to `super` should have a newline after it